### PR TITLE
refactor(test): apply simpler test structure

### PR DESCRIPTION
**Pull request by Djaytan**

### CODE REVIEW

File: patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/ConfigValidatingPropertiesTest.java [hunk 1]
```diff
@@ -25,12 +25,15 @@
 package fr.djaytan.minecraft.jobsreborn.patchplacebreak.config.annotated;
```
Comment (Djaytan) [inline patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/ConfigValidatingPropertiesTest.java:25]: A merge between ConfigValidatingPropertiesTest and ConfigValidatingPropertiesIT

File: patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/ConnectionPoolValidatingPropertiesTest.java [hunk 2]
```diff
@@ -25,9 +25,10 @@
 package fr.djaytan.minecraft.jobsreborn.patchplacebreak.config.annotated;
```
Comment (Djaytan) [inline patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/ConnectionPoolValidatingPropertiesTest.java:25]: A merge between ConnectionPoolValidatingPropertiesTest and ConnectionPoolValidatingPropertiesIT

File: patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/CredentialsValidatingPropertiesTest.java [hunk 3]
```diff
@@ -25,9 +25,10 @@
 package fr.djaytan.minecraft.jobsreborn.patchplacebreak.config.annotated;
```
Comment (Djaytan) [inline patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/CredentialsValidatingPropertiesTest.java:25]: A merge between CredentialsValidatingPropertiesTest and CredentialsValidatingPropertiesIT

File: patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/DataSourceValidatingPropertiesTest.java [hunk 4]
```diff
@@ -25,9 +25,10 @@
 package fr.djaytan.minecraft.jobsreborn.patchplacebreak.config.annotated;
```
Comment (Djaytan) [inline patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/DataSourceValidatingPropertiesTest.java:25]: A merge between DataSourceValidatingPropertiesTest and DataSourceValidatingPropertiesIT

File: patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/DbmsHostValidatingPropertiesTest.java [hunk 5]
```diff
@@ -25,7 +25,9 @@
 package fr.djaytan.minecraft.jobsreborn.patchplacebreak.config.annotated;
```
Comment (Djaytan) [inline patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/DbmsHostValidatingPropertiesTest.java:25]: A merge between DbmsHostValidatingPropertiesTest and DbmsHostValidatingPropertiesIT

File: patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/DbmsServerValidatingPropertiesTest.java [hunk 6]
```diff
@@ -25,7 +25,9 @@
 package fr.djaytan.minecraft.jobsreborn.patchplacebreak.config.annotated;
```
Comment (Djaytan) [inline patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/DbmsServerValidatingPropertiesTest.java:25]: A merge between DbmsServerValidatingPropertiesTest and DbmsServerValidatingPropertiesIT

**Review by Niiz9 — DISMISSED** [review]

**Pull request merged**